specs/agents/tracing-spans.md

```diff
@@ -111,7 +101,7 @@ However, when tracing a regular outgoing HTTP request (one that's not initiated
 and it's unknown whether the downsteam service continues the trace,
 the trace headers should be added.
 
-The reason is that spans cannot be compressed (TODO link to span compression spec once available) if the context has been propagated, as it may lead to orphaned transactions.
+The reason is that spans cannot be [compressed](tracing-spans-compress.md) if the context has been propagated, as it may lead to orphaned transactions.
```

Comment:
```suggestion
The reason is that spans cannot be [compressed](handling-huge-traces/tracing-spans-compress.md) if the context has been propagated, as it may lead to orphaned transactions.
```